selenebuild/selenebuild.moon

```diff
@@ -0,0 +1,209 @@
+-- Selene Build Script
```

Comment:
do you see adding more files in selenebuild/ in the future? If not may as well keep this file at the root to keep paths shorter, i.e. /selenebuild.moon or even /build.moon